process/release_process.md

```diff
@@ -0,0 +1,155 @@
+# Release process
+
+Document version: **v1.0.0**
+
+Document status: **Released**
+
+## Table of Contents
+
+- [Overview](#overview)
+- [Impacted domains](#impacted-domains)
+- [How to create a release?](#how-to-create-a-release)
+  - [xt-core](#xt-core)
+  - [moulin-based products](#moulin-based-products)
+- [Split of responsibilities](#split-of-responsibilities)
+- [Example of releasing the moulin-based project](#example-of-releasing-the-moulin-based-project)
+
+Please note that this table of contents is written in Markdown and will work best in a markdown-friendly environment. Hyperlinks in the table of content won't work here. They need to be associated with the actual sections in your markdown document. The text after the '#' should match exactly the title of the section.
+
+## Overview
+
+This document describes release creation rules for the moulin development environment.
+
+The further part of the document is based on the Edge Orchestrator reference project:
+
+![Edge Orchestrator project high-level component diagram](./res/ref_project.png) 
+
+**Note!** Do not worry that a specific example is used below. The conclusions from this example have a broader context and can also be applied to other moulin-based products.
+
+## Impacted Domains
+
+Each component we develop in the context of the moulin builds should contain releases. It means that every domain has to freeze their code from time to time:
+
+- repositories with moulin configuration *.yaml files - should have their releases
+- repositories with repo manifests, west manifests, and bit-bake recipes - should have their releases
+- software component code repositories referenced by all above configurations - should have their releases
+
+But let's think of **higher levels of abstraction** and use more significant entities than git repositories.
+
+In our current practical scenario, we have an Edge Orchestrator reference project that consists of the following high-level components:
+
+- **xt-core** - a set of repositories inside the xen-troops GitHub organization that contains the core build functionality mainly as Yocto layers.
+  Note! It is up to xen-troops maintainers to define which repositories will be included in the xt-core abstraction. Currently, the most obvious candidates are listed below:
+  https://github.com/xen-troops/meta-xt-common
+  https://github.com/xen-troops/meta-xt-rcar
+  https://github.com/xen-troops/meta-xt-vhost

```

Comment:
Please add `-` before each repo, so that the rendered text will keep the required formatting.

Reply:
Hi @rshym, done.